editor/script/editor_script.cpp

```diff
@@ -65,12 +65,8 @@ void EditorScript::add_root_node(Node *p_node) {
 }
 
 Node *EditorScript::get_scene() const {
-	if (!EditorNode::get_singleton()) {
-		EditorNode::add_io_error("EditorScript::get_scene: " + TTR("Write your logic in the _run() method."));
-		return nullptr;
-	}
-
-	return EditorNode::get_singleton()->get_edited_scene();
+	WARN_PRINT_ONCE("EditorScript::get_scene is deprecated. Use EditorInterface::get_edited_scene_root instead.");
```

Comment:
```suggestion
	WARN_DEPRECATED_MSG("EditorScript::get_scene is deprecated. Use EditorInterface::get_edited_scene_root instead.");
```
Any reason to not use the dedicated macro for this?

Reply:
Of course, the one example I looked at `WARN_PRINT_ONCE("instance_set_interpolated() is deprecated.");`  uses the wrong one lol, thanks for point this out.